buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/publish/PublishingPlugin.groovy

```diff
@@ -111,18 +114,13 @@ class PublishingPlugin implements Plugin<Project> {
                                 artifactId = "$sub.parent.name-ktx"
                                 groupId = sub.parent.group
                             }
-                            pom {
-                                licenses {
-                                    license {
-                                        name = 'The Apache Software License, Version 2.0'
-                                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
-                                    }
-                                }
-                                scm {
-                                    connection = 'scm:git:https://github.com/firebase/firebase-android-sdk.git'
-                                    url = 'https://github.com/firebase/firebase-android-sdk'
+                            if (firebaseLibrary.publishSources) {
```

Comment:
Thinking out loud:
While the pom files we generate here will link to sources, we are still dependent on our internal release process to publish sources, which may not be supported yet.

Reply:
correct, it still needs to get plumbed through explicitly